lua/wire/gates/entity.lua

```diff
@@ -17,10 +17,11 @@ GateActions["entity_applyf"] = {
 	inputtypes = { "ENTITY" , "VECTOR" },
 	timed = true,
 	output = function(gate, ent, vec )
+               if not ApplyForceValidAction(gate) then return end
```

Comment:
You're passing the gate here though in the E2 you're passing the context. I'd say pass the player and the entity being forced.